stdlib/asm/crypto/stark/deep_queries.masm

```diff
@@ -480,19 +495,18 @@ export.compute_deep_composition_polynomial_queries
         # Load the (main, aux, constraint)-traces rows associated with the current query and get
         # the index of the query.
         #
-        # Cycles: 219
+        # Cycles: 206
         exec.load_query_row
-        #=>[index, query_ptr, query_end_ptr, ...]
+        #=>[Y, Y, index, query_ptr, query_end_ptr, ...]
```

Comment:
I would add a comment that `X` and `Y` are "garbage" values.

One general thought: should we always use `X` for garbage values? (i.e., instead of sometimes using `X` and sometimes `Y`).

Reply:
Added and switched all garbage values to be either `Y` for words or `y` for elements. This is better as we use `x` for other things in the code